docs/reference/commandline/daemon.md

```diff
@@ -715,10 +715,9 @@ the first 65536 range:
      dockremap:165536:65536
 ```
 
-> **Note:** On a fresh Fedora install, we had to `touch` the
-> `/etc/subuid` and `/etc/subgid` files to have ranges assigned when users
-> were created.  Once these files existed, range assignment on user creation
-> worked properly.
+> **Note:** On Fedora < 23, you have to `touch` the `/etc/subuid` and `/etc/subgid`
```

Comment:
Perhaps "On Fedora 22 and older" (or just "Fedora 22", because we no longer build for "Fedora 21"